helm_deploy/values-dev.yaml

```diff
@@ -5,8 +5,6 @@ generic-service:
   replicaCount: 2
   ingress:
     host: registers-to-delius-update-dev.hmpps.service.justice.gov.uk
-    contextColour: green
```

Comment:
We usually leave an empty line between `generic-service.ingress` and `generic-service.env` but it's not a huge issue